1-exercises/B-setting-values/exercise1.js

```diff
@@ -23,6 +23,11 @@ let capitalCities = {
 */
 
 // write code here
+capitalCities.UnitedKingdom.population = 8980000;
+capitalCities.China.population = 21500000;
+capitalCities.Peru = {}
+capitalCities.Peru.name = "Lima"
+capitalCities.Peru.population = 9750000;
```

Comment:
this works well done, you can also set the properties of Peru (name & population) when you create the object:
`capitalCities.Peru = {
  name: "Lima",
  population: 9750000,
};`